File: JavaScript/1019 - Conversão de Tempo.js
var N = parseInt(require('fs').readFileSync('/dev/stdin', 'utf8'));

var horas = Math.floor(N / 3600);
N %= 3600;
var minutos = Math.floor(N / 60);
var segundos = N % 60;

console.log(`${horas}:${minutos}:${segundos}`);

File: Clojure/1019 - Conversão de Tempo.clj
(def tempo-total (read))

(def horas (int (/ tempo-total 3600)))
(def resto-hora (mod tempo-total 3600))
(def minutos (int (/ resto-hora 60)))
(def segundos (mod resto-hora 60))

(printf "%d:%d:%d\n" horas minutos segundos)